frontend/src/_includes/table_reports.liquid

```diff
@@ -2,19 +2,9 @@
   <div class="grid-col-12">
     <h2>Test results</h2>
 
-    <table class="usa-table usa-table--borderless prime-table" summary="Previous results">
-      <thead>
-        <tr>
-          <th scope="col">Report Id</th>
-          <th scope="col">Date Sent</th>
-          <th scope="col">Expires</th>
-          <th scope="col">Total tests</th>
-          <th scope="col">File</th>
-        </tr>
-      </thead>
-      <tbody id="tBody" class="font-mono-2xs">
-      </tbody>
-    </table>
+    <!-- <ul id="reportFeeds" class="usa-button-group usa-button-group--segmented"></ul> -->
```

Comment:
Remove the table scaffolding construction (and move it to the js)  replace with tab wrapper